node/metrics/auth.ts

```diff
@@ -16,6 +16,7 @@ export interface AuthAuditMetric {
   hasValidStoreToken?: boolean
   hasApiToken?: boolean
   hasValidApiToken?: boolean
+  hasAdminTokenOnHeader?: boolean
```

Comment:
Just to reenforce, I believe adding metrics specific for the OnHeader use case will make it easier to understand the metrics (i.e. also add `hasValidAdminTokenOnHeader`)

Reply:
Done